src/elements/common/progress-bar/ProgressBar.js

```diff
@@ -112,6 +113,7 @@ class ProgressBar extends PureComponent<Props, State> {
     render() {
         const { percent }: State = this.state;
         const containerStyle = {
+            zIndex: overlayZIndex,
```

Comment:
Progress bar could get hidden behind grid cards when scrolling.  Is this a fix or a hack?

Reply:
Can the z-index be set on the `.be-progress-container` class, instead?